### ISSUE TITLE
Uncaught TypeError: Cannot read property 'endsWith' of undefined

### ISSUE THREAD

**Issue by sattvikc** [aeikenberry/atom-python-package#3]
[Enter steps to reproduce:]

Happens randomly

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Ubuntu 17.04
**Thrown From**: [atom-python-package](https://github.com/aeikenberry/atom-python-package) package 0.7.0

### Stack Trace

Uncaught TypeError: Cannot read property 'endsWith' of undefined

```
At /home/sattvik/.atom/packages/atom-python-package/lib/atom-python-package.js:94

TypeError: Cannot read property 'endsWith' of undefined
    at Object.setDir (/packages/atom-python-package/lib/atom-python-package.js:94:14)
    at Object.toggleAtDir (/packages/atom-python-package/lib/atom-python-package.js:89:10)
    at HTMLLIElement.atomPythonPackageToggleAtDir (/packages/atom-python-package/lib/atom-python-package.js:37:58)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/usr/share/atom/resources/app/src/atom-environment.js:1320:34)
    at EventEmitter.outerCallback (/usr/share/atom/resources/app/src/application-delegate.js:347:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -9:41.7.0 editor:newline (input.hidden-input)
     -9:41.5.0 intentions:highlight (input.hidden-input)
     -9:41.2.0 core:paste (input.hidden-input)
     -9:40.7.0 core:save (input.hidden-input)
 11x -6:38.5.0 core:confirm (span.github-FilePatchListView-path)
     -0:36.2.0 core:close (div.github-StagingView.unstaged-changes-focused)
     -0:34.1.0 intentions:highlight (input.hidden-input)
     -0:33.9.0 core:close (input.hidden-input)
     -0:24.7.0 atom-python-package:toggleAtDir (div.header.list-item)
     -0:16.4.0 core:cancel (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
     -0:13.1.0 atom-python-package:toggleAtDir (div.header.list-item)
  2x -0:12.4.0 core:cancel (div.tool-panel.tree-view)
     -0:11.2.0 core:save (atom-pane.pane)
     -0:08.4.0 atom-python-package:toggleAtDir (div.header.list-item)
```

### Non-Core Packages

```
apex-syntax 0.1.0 
apex-ui 0.2.1 
atom-jinja2 0.6.0 
atom-material-syntax 1.0.6 
atom-material-ui 2.0.4 
atom-python-package 0.7.0 
autoclose-html 0.23.0 
busy-signal 1.4.3 
chester-atom-syntax 0.2.0 
file-icons 2.1.8 
fonts 3.0.2 
git-diff-details 1.4.0 
git-plus 7.9.3 
git-time-machine 1.5.9 
intentions 1.1.2 
language-babel 2.67.0 
language-cmake 0.3.0 
language-cython 0.3.0 
language-docker 1.1.8 
language-restructuredtext 1.1.0 
linter 2.2.0 
linter-docker 0.2.0 
linter-python 3.1.2 
linter-ui-default 1.6.2 
material 2.2.0 
python-debugger 0.2.0 
python-isort 0.0.7 
python-jedi 0.3.8 
python-snippets 1.9.0 
seti-syntax 1.1.3 
seti-ui 1.9.0 
split-diff 1.4.3 
tool-bar 1.1.0 
```

**Comment by aeikenberry** [comment]
Thanks for reporting. I haven't used Atom in a long while, but happy to fix if you're using this! v0.7.1 is up!

**Comment by sattvikc** [comment]
Thank you for the fix, working perfectly now!
